src/mlpack/core/data/tokenizer.hpp

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
Probably, it's a good idea to move the file to some folder like `src/mlpack/core/data/tokenizers/`.

Reply:
Done.